config/capi-rinkeby.py

```diff
@@ -0,0 +1,33 @@
+from core.aws import get_secrets
+
+DEVELOPMENT = True
```

Comment:
i think later we can ditch references to `DEVELOPMENT` in favor of `MAINNET` or `!MAINNET` etc... and remove the old development and staging files, but it can wait...

technically skynet and rinkeby _are_ "development" but i'm just being OCD as usual

fine for now